test/add_procs_threading.jl

```diff
@@ -0,0 +1,46 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+using Base.Test
+
+type MLKNotImplementedException <: Exception end
+
+"""
+    get_num_threads(n)
+
+Get the number of threads the BLAS library is using.
+"""
+function get_num_threads()
```

Comment:
Only the set_num_threads versions is wrapped there currently.  I could add this there, though mkl doesn't support it. Would it make sense to do that in a separate PR?